compaction/time_window_compaction_strategy.cc

```diff
@@ -36,6 +36,10 @@ const std::unordered_map<sstring, std::chrono::seconds> time_window_compaction_s
 const std::unordered_map<sstring, time_window_compaction_strategy_options::timestamp_resolutions> time_window_compaction_strategy_options::valid_timestamp_resolutions = {
     { "MICROSECONDS", timestamp_resolutions::microsecond },
     { "MILLISECONDS", timestamp_resolutions::millisecond },
+    { "SECONDS",      timestamp_resolutions::seconds },
+    { "MINUTES",      timestamp_resolutions::minutes },
+    { "HOURS",        timestamp_resolutions::hours },
+    { "DAYS",         timestamp_resolutions::days },
```

Comment:
A few lines above these lines there is a valid_window_units which already lists MINUTES, HOURS and DAYS. So I think (but we need to check) there's no need to add them again here. I think you can add SECONDS to that valid_window_units. I don't know why we have this extra "valid_timestamp_resolutions" but I think it was created just for sub-second units :-(